products/developertoolset/get-started-rhel7-cpp.adoc

```diff
@@ -1,7 +1,15 @@
 :awestruct-layout: product-get-started-multipath
 :awestruct-interpolate: true
-:title: "Developer Toolset - Get started developing with C++ and Eclipse on RHEL 7"
-:awestruct-description: "Get started developing with C++ and Eclipse from Red Hat Developer Toolset on Red Hat Enterprise Linux 7"
+:tthw-site-base-url: #{site.base_url}
+:tthw-cdn-site-base-url: #{cdn(site.base_url)}
```

Comment:
Yes, that's the theory, what's different, I think, is the CDN bits. 

 The complaints that I found in the build log file are related to images.  The lines you highlighted are the same trick being used to set the base urls for the site and for download manager. 

I'm seeing 4 bad URI errors and 6 Error cdn-ifying errors. all related to images.

 /usr/local/bin/bundle:23:in `<main>'
13:33:24 Error cdn-ifying img <img src="#%7Bcdn(site.base_url%20+%20'/images/products/multipath/cpp-logo.png')%7D" alt="Eclipse CPP Logo" title="" developer toolset get started developing with c and eclipse on rhel>

and 

An error occurred: bad URI(is not URI?): images/products/softwarecollections/rhel{tthw-rhelver}/manage-repositories.png